src/platform/utilities/ui/focus.js

```diff
@@ -61,21 +61,33 @@ export function focusElement(selectorOrElement, options, root) {
  * iterations, then fall back to the default selector (step _ of _ h2)
  * Discussion: https://dsva.slack.com/archives/CBU0KDSB1/p1676479946812439
  * @param {String} selector - focus target selector
- * @param {Element} root - starting element of the querySelector; may be a
- *  shadowRoot
+ * @param {Element} root - starting element of the querySelector
+ * @param {Number} timeInterval - time in milliseconds to delay
+ * @param {String} internalSelector - selector pointing to an element inside the
+ *  component we're waiting for (could be an element in shadow DOM)
  * @example waitForRenderThenFocus('h3', document.querySelector('va-radio').shadowRoot);
  */
 export function waitForRenderThenFocus(
   selector,
   root = document,
   timeInterval = 250,
+  // added because we first need to wait for a component to be rendered, then we
+  // need to target an element inside the component (in regular or in a web
+  // component's shadow DOM)
+  internalSelector,
```

Comment:
since this doesn't have a default value, will this function error if it's called without the argument passed in?

Reply:
It won't. Line 86 checks for a truthy value before it's used.

Reply:
ah okay cool, just had to double check since i've been in Ruby land for a while 😄 thanks!